spacy/training/example.pyx

```diff
@@ -1,5 +1,5 @@
 from collections.abc import Iterable as IterableInstance
-import numpy
+import numpy as np
```

Comment:
Let's not change this. The codebase consistently uses:

- `cimport numpy as np`
- `import numpy`